test/test_serialization.py

```diff
@@ -747,6 +748,7 @@ def run(self, *args, **kwargs):
         with serialization_method(use_zip=True):
             return super(TestSerialization, self).run(*args, **kwargs)
 
+instantiate_device_type_tests(TestBothSerialization, globals())
```

Comment:
I'm surprised this didn't cause failures with the inherited tests from the mixin.